src/Maker/MakeRegistrationForm.php

```diff
@@ -209,17 +212,36 @@ private function interactAuthenticatorQuestions(InputInterface $input, ConsoleSt
 
         $this->firewallName = $firewallName;
 
-        // get list of guard authenticators
-        $authenticatorClasses = $interactiveSecurityHelper->getAuthenticatorClasses($firewallsData[$firewallName]);
-        if (empty($authenticatorClasses)) {
-            $io->note('No Guard authenticators found - so your user won\'t be automatically authenticated after registering.');
-        } else {
-            $this->autoLoginAuthenticator =
-                1 === \count($authenticatorClasses) ? $authenticatorClasses[0] : $io->choice(
-                    'Which authenticator\'s onAuthenticationSuccess() should be used after logging in?',
-                    $authenticatorClasses
-                );
+        // get list of authenticators
+        $authenticators = $interactiveSecurityHelper->getAuthenticatorClasses($firewallsData[$firewallName]);
+
+        $canLoginAfterRegistration = $this->canSecurityBundleLogin;
+
+        if (!$this->canSecurityBundleLogin) {
+            foreach ($authenticators as $authenticator) {
+                if (!$authenticator->isNative()) {
```

Comment:
I don't like this `isNative()`: we're hardcoding valid "authenticators". Can't we allow the user to choose from *all* possible if there are multiple? Could we get the list of "authenticator names" the same way that the `Security` class does - e.g. however this `authenticators` property is set?